### PR TITLE
Create an Army document type (No sheet)

### PR DESCRIPTION
**Pull request by silvative**

### CODE REVIEW

**Comment by silvative** [comment]
Hmm, I think Git is recognising the moved files as new files instead of updated files, which is why it says there's an ID conflict. What's the best way to resolve this?

File: static/lang/kingmaker-en.json [hunk 1]
```diff
@@ -319,5 +319,141 @@
...
+                "warning": "Warning: The old statistics will be permanently overwritten.",
+                "parametersHeader": "Parameters",
+                "levelLabel": "Level: ",
+                "saveLabel": "Strong save: ",
+                "confirmButton": "Update Stats",
+                "levelUpTitle": "Level Up Army",
+                "levelUpDesc": "Increases Army level by 1, updating stats using the default values as defined in the Warfare rules. New statistics will be dependent on the level. Gear, tactics, actions, and traits will not be replaced."
+            }
+        },
+        "Traits": {
```
Comment (CarlosFdez) [inline static/lang/kingmaker-en.json:383]: Move these traits to `PF2E.Kingmaker.Trait`, and create a new `PF2E.Kingmaker.TraitDescription` object. `armyDescription` would then be `PF2E.Kingmaker.TraitDescription.army` instead.

**Comment by CarlosFdez** [comment]
Doing the ranks the way listed is pretty verbose and I don't think its necessary. Instead make the ranks a number property listing the number of ranks, and the rank idx to fill `PF2E.Warfare.Gear.Weapons.rank${idx}.name` in the sheet itself.

File: src/module/actor/army/data.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    BaseActorSourcePF2e,
+    BaseHitPointsSource,
+} from "@actor/data/base.ts";
+import { ARMY_TYPES } from "./values.ts";
+import { ActorSizePF2e } from "@actor/data/size.ts";
+
+type ArmySource = BaseActorSourcePF2e<"army", ArmySystemSource>;
+
+interface ArmySystemSource extends ActorSystemSource {
+    attributes: ArmyAttributesSource;
```
Comment (CarlosFdez) [inline src/module/actor/army/data.ts:21]: Attribute is on the chopping block because of remaster, though I don't know what @stwlam plans to do yet.

Comment (stwlam) [reply src/module/actor/army/data.ts:21]: We still need to get everything out of there, so it's fine for now.

**Pull request closed without merging**